mathml.css

```diff
@@ -19,8 +19,15 @@ math[display="block"] {
 /* fraction */
 mfrac {
     display: inline-block !important;
-    vertical-align: -50%;
+    vertical-align: 30%;
     border-collapse: collapse;
+    line-height: 110%;
+    font-size: 90%;
+}
+@media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
+    mfrac {
+        vertical-align: -50%;
+    }
```

Comment:
@kirilloid , It seems, it will work for you, if you simply change `vertical-align: top;` to `vertical-align: text-top;`.